learn/get-started/install-ballerina/Installation-options.md

```diff
@@ -99,6 +99,10 @@ Language specification 2022R1
 update tool 1.3.8
 ```
 
+## Build from source
+
+For instructions, see [Build from source](https://github.com/ballerina-platform/ballerina-distribution/blob/master/docs/build-from-source.md).
```

Comment:
```suggestion
For instructions, see [Build from source](github.com/ballerina-platform/ballerina-distribution/blob/master/docs/building-from-source.md).
```

Reply:
This [file](https://github.com/ballerina-platform/ballerina-distribution/tree/master/docs) is renamed now via https://github.com/ballerina-platform/ballerina-distribution/pull/2880.